website/templates/report.html

```diff
@@ -367,6 +381,24 @@ <h2 class="text-2xl font-semibold leading-7 text-gray-900">{% trans "ADD TEAM ME
 {% endblock content %}
 {% block after_js %}
     <script type="text/javascript">
+    function isCVEValid(cveNumber) {
+        const cveRegex = /^CVE-\d{4}-\d{4,}$/;
```

Comment:
It should be limited from the other side too
```suggestion
        const cveRegex = /^CVE-\d{4}-\d{4,7}$/;
```

Reply:
done 👍